x-pack/plugins/canvas/storybook/storyshots.test.tsx

```diff
@@ -94,6 +95,12 @@ jest.mock('../shareable_runtime/components/rendered_element');
 // @ts-expect-error
 RenderedElement.mockImplementation(() => 'RenderedElement');
 
+// Some of the code requires that this directory exists, but the tests don't actually require any css to be present
+const cssDir = path.resolve(__dirname, '../../../../built_assets/css');
```

Comment:
@clintandrewhall FYI, I'm removing the legacy sass stuff soon so this won't stick around for long